pkgs/development/androidndk-pkgs/default.nix

```diff
@@ -50,9 +50,7 @@
       inherit lib;
       inherit (buildPackages)
         makeWrapper;
-      inherit (pkgs)
-        stdenv
-        runCommand wrapBintoolsWith wrapCCWith;
+      inherit pkgs;
```

Comment:
Making the same change to the `18b` expressoin above will fix the ofborg error

Reply:
Friendly ping - are you able to fix the eval error?

Reply:
sorry it took so long. Fixed now.

Reply:
No worries at all, wasn't sure you got the initial suggestion. Thanks for the fix!